src/Error/ExceptionRenderer.php

```diff
@@ -183,7 +183,6 @@ public function render()
                 'format' => 'array',
                 'args' => false
             ]);
-            $viewVars['_serialize'][] = 'trace';
```

Comment:
As before though, they sometimes cause fatal errors. The stack trace is still available in the logs.